Command/SshCommand.php

```diff
@@ -84,6 +90,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $username = $input->getArgument('username');
         $service = $input->getOption('service');
         $bindAddress = $input->getOption('bind-address');
+        $toolsDb = $input->hasOption('toolsdb');
```

Comment:
`hasOption` just reports whether the option *exists*, so is always true. If you set it to VALUE_NONE above, and then here use `getOption` it'll be a boolean.

Reply:
Thank you! Now I know how it works :) I made `bind-address` have `VALUE_REQUIRED` as that's what it should be. Hope it's okay to slip that into the same PR.